internal/common/objects/BaseBiker.go

```diff
@@ -181,6 +197,40 @@ func (bb *BaseBiker) GetResourceAllocationParams() ResourceAllocationParams {
 	return bb.allocationParams
 }
 
+// default implementation returns the id of the nearest lootbox
+func (bb *BaseBiker) ProposeDirection() uuid.UUID {
+	return bb.NearestLoot()
+}
```

Comment:
I don't think that returning an ID is the best option, for posterity. 

It locks users into having to specify an ID for their direction, when they may want to avoid the Audi, say, by choosing some arbitrary direction. 

It's probably best to have this as a coordinate or, even better, a vectorised bearing